src/github.rs

```diff
@@ -0,0 +1,93 @@
+use curl;
```

Comment:
Could you add a doc comment here to give a high level overview of what this module does and where it's used?